library/std/src/sys/pal/wasi/thread.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Can this change to `forbid` and move to the parent `library/std/src/sys/pal/wasi/mod.rs`?